### PR TITLE
Centralized languages into 1 files

### PR DESCRIPTION
**Pull request by ZeroX-DG**
This PR centralized all languages into 1 file so that next time you want to add a new language just add it to the Languages.js file.

### CODE REVIEW

File: browser/main/modals/PreferencesModal/UiTab.js [hunk 1]
```diff
@@ -182,22 +183,11 @@ class UiTab extends React.Component {
...
-                <option value='ja'>{i18n.__('Japanese')}</option>
-                <option value='ko'>{i18n.__('Korean')}</option>
-                <option value='no'>{i18n.__('Norwegian')}</option>
-                <option value='pl'>{i18n.__('Polish')}</option>
-                <option value='pt-BR'>{i18n.__('Portuguese (Brazil)')}</option>
-                <option value='pt-PT'>{i18n.__('Portuguese (Portugal)')}</option>
-                <option value='ru'>{i18n.__('Russian')}</option>
-                <option value='es-ES'>{i18n.__('Spanish')}</option>
+                {
+                  languages.map((language) => {
```
Comment (sosukesuzuki) [inline browser/main/modals/PreferencesModal/UiTab.js]: `()` for a single argument and `return` are unnecessary in arrow functions.
please rewrite like below:
```jsx
languages.map(language =>
  <option value={language.locale} key={language.locale}>{i18n.__(language.name)}</option>)
```

File: browser/lib/i18n.js [hunk 2]
```diff
@@ -1,11 +1,17 @@
 const path = require('path')
 const { remote } = require('electron')
 const { app } = remote
+const { languages } = require('./Languages.js')
+
+let locales = languages.reduce(function (localeList, locale) {
```
Comment (sosukesuzuki) [inline browser/lib/i18n.js]: `locales` is never reassigned. Please use `const` instead of `let`.

Comment (sosukesuzuki) [reply browser/lib/i18n.js]: And this duplicates with [this lines](https://github.com/ZeroX-DG/Boostnote/blob/65d8d7282f8f8fad5ff23a31b432a5a97d222a6f/browser/main/Main.js#L156-L159). 
Please export `locales` from this file and import `locales` from this file in `browser/main/Main.js`.

File: browser/main/Main.js [hunk 3]
```diff
@@ -152,24 +153,12 @@ class Main extends React.Component {
...
-      'hu',
-      'ja',
-      'ko',
-      'no',
-      'pl',
-      'pt',
-      'ru',
-      'es-ES'
-    ]
+    let locales = languages.reduce(function (localeList, locale) {
```
Comment (sosukesuzuki) [inline browser/main/Main.js]: please confirm [this review](https://github.com/BoostIO/Boostnote/pull/1818#discussion_r183685530)

**Comment by ZeroX-DG** [comment]
@sosukesuzuki please confirm my changes

**Comment by sosukesuzuki** [comment]
Good! I'll approve!!:+1:

**Comment by kazup01** [comment]
Merged. Thank you for amazing contribution @ZeroX-DG !

**Pull request merged**